pdp_util/filters.py

```diff
@@ -64,63 +69,60 @@ def validate(self, value):
         return value if m else None
 
     def __str__(self):
-        return '<FormFilter> %s:%s' % (self.name, self.value)
+        return "<FormFilter> %s:%s" % (self.name, self.value)
 
 
 def mk_mp_regex():
-    decimal = r'-?[0-9]+(.[0-9]+)?'
-    point = '%(decimal)s %(decimal)s' % locals()
-    inner = r'\(%(point)s(, ?%(point)s){2,}\)' % locals()
+    decimal = r"-?[0-9]+(.[0-9]+)?"
+    point = "%(decimal)s %(decimal)s" % locals()
+    inner = r"\(%(point)s(, ?%(point)s){2,}\)" % locals()
```

Comment:
f-string?

Reply:
Can I do that with a regular expression?

Reply:
Oh sorry I didn't notice some of these were regex.